js/contextMenus.js

```diff
@@ -501,6 +505,19 @@ function mainTemplateInit (nodeProps, frame) {
       label: 'Add to reading list',
       enabled: false
     })
+
+  template.push(CommonMenu.separatorMenuItem)
+  
+  template.push({
+    label: 'Inspect Element',
+    click: (item, focusedWindow) => {
+      if (focusedWindow) {
+        document.querySelector( 'webview' ).openDevTools();
+        document.querySelector( 'webview' ).inspectElement(nodeProps.offsetX, nodeProps.offsetY);
```

Comment:
This is looking a lot better now thanks for cleaning it up. 
Couple of minor suggestions:
i) create a `js/actions/windowAction.js` and put this code in there.  Call it something like inspectElement (x, y).  
ii) Only query the dom element once and store it in a variable, and then do an if (webview) {} in case the dom element doesn't exist.   Also I think you may only need the inpsectElement and not the openDevTools() but whatever is needed is fine.